packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```diff
@@ -673,5 +672,13 @@
   "ip_table_manage_delegation_ipv6block_delete_err": "Une erreur s'est produite lors de la suppression de la délégation de reverse DNS :",
   "ip_table_manage_delegation_ipv6block_resume": "Résumé de l'opération",
   "ip_table_manage_delegation_ipv6block_to_add": "Reverse(s) à ajouter",
-  "ip_table_manage_delegation_ipv6block_to_delete": "Reverse(s) à supprimer"
+  "ip_table_manage_delegation_ipv6block_to_delete": "Reverse(s) à supprimer",
+  "ip_order_title": "Commander",
+  "ip_order_additional_ips": "IP supplémentaires",
+  "ip_order_byoip": "Bring your own IP",
```

Comment:
To be sure, the PU need it in english, right ?

Reply:
Yes, 'Bring you own IP' is a product name. So no translation is required.

Reply:
**question:** If it's not meant to be translated, why adding it into translation file then?